examples/dem/3d-bunny-drill/bunny-drill-loading.prm

```diff
@@ -91,14 +91,14 @@ end
 #---------------------------------------------------
 
 subsection insertion info
-  set insertion method                               = volume
-  set inserted number of particles at each time step = 10000
-  set insertion frequency                            = 100000
-  set insertion box points coordinates               = 0.1, -0.07, -0.07 : 0.24, 0.07, 0.07
-  set insertion distance threshold                   = 1.10
-  set insertion maximum offset                       = 0.10
+  set insertion method                               = plane
```

Comment:
This is a great example for the feature, this is also cool to watch the particle spawn :D